testing/libgeotiff/APKBUILD

```diff
@@ -31,4 +33,15 @@ package() {
 	make DESTDIR="$pkgdir" install
 }
 
-sha512sums="059c6e05eb0c47f17b102c7217a2e1636e76d622c4d1bdcf0bd89fb3505f3130bffa881e21c73cfd2ca0d6863b81322f85784658ba3539b53b63c3a8f38d1deb  libgeotiff-1.4.2.tar.gz"
+check() {
+	bin/makegeo
+	bin/listgeo newgeo.tif > metadata.txt
+	grep "Ellipsoid: 7016/Everest 1830 (1967 Definition) (6377298.56,6356097.55)" metadata.txt
```

Comment:
Can we get a comment that explains what this is about ? i can't make out what this is all supposed to mean.

Reply:
added some comments where I try to explain the test, which is from https://github.com/OSGeo/libgeotiff#building-libgeotiff